src/odemis/driver/technolution.py

```diff
@@ -368,69 +435,92 @@ def _acquire(self):
+            self.parent.ASM_API_Post_Call("/scan/finish_mega_field", 204)
             logging.debug("Acquisition thread ended")
 
     def start_acquisition(self):
+        """
+        Put a the command 'start' mega field scan on the queue with the appropriate MegaFieldMeta Model of the mega
+        field image to be scannend. All subsequent calls to scan_field will use a part of this meta data to store the image
+        data until the stop command is executed.
+        """
+        if not self._acq_thread or not self._acq_thread.is_alive():
+            logging.info('Starting acquisition thread and clearing remainder of the old queue')
+
+            # Clear the queue
+            while True:
+                try:
+                    self.acq_queue.get(block=False)
+                except queue.Empty:
+                    break
+
+            self._acq_thread = threading.Thread(target=self._acquire,
+                                                name="acquisition thread")
+            self._acq_thread.deamon = False
+            self._acq_thread.start()
+
         megafield_metadata = self._assemble_megafield_metadata()
         self.acq_queue.put(("start", megafield_metadata))
 
     def get_next_field(self, field_num):
         '''
+        Put a the command 'next' field image scan on the queue with the appropriate field meta data model of the field
+        image to be scannend. Can only be executed if it proceeded by a 'start' mega field scan command on the queue.
+        As notifier function the dataflow.notify is given which means the returned image will be redirected to this
+        function.
+
         :param field_num: x,y
-        :return:
         '''
-        self._field_data = FieldMetaData(
-                field_num[0] * self.parent._ebeam_scanner.resolution.value[0],
-                field_num[1] * self.parent._ebeam_scanner.resolution.value[1])
-        #TODO K.K. remove temporary vas A
-        A = self.convert_field_num2pixels(field_num)
-        self.acq_queue.put(("next", self._field_data, self.dataContent.value, self.dataFlow.notify))
+        field_data = FieldMetaData(*self.convert_field_num2pixels(field_num))
+        self.acq_queue.put(("next", field_data, self.dataContent.value, self.data.notify))
 
     def stop_acquisition(self):
-        self.acq_queue.put(("stop", ))
+        """
+        Puts a 'stop' field image scan on the queue, after this call, no fields can be scanned anymore. A new mega
+        field can be started. The call triggers the post prosessing process to generate and offload additional zoom
+        levels
+        """
+        self.acq_queue.put(("stop",))
 
     def acquire_single_field(self, field_num=(0, 0)):
+        """
+        Puts a the series 'start','next','stop' commands on the queue with the appropriate metadata models and
+        scans a single field image. By providing as notifier function a return_queue the image can be returned. The
```

Comment:
I don't understand the second sentence. 
The sentence on the queue is a general info on the queue that does not belong in the doc string.